transform/mattermost-analytics/models/staging/mm_telemetry_prod/_mm_telemetry_prod__models.yml

```diff
@@ -28,67 +28,43 @@ models:
       Reconstructed `performance_events` table from schema `MM_TELEMETRY_PROD` using only columns needed by looker.
 
     columns:
-      - name: context_page_referrer          
-      - name: channel                        
-      - name: context_app_namespace          
-      - name: id                             
-      - name: user_actual_id                 
-      - name: context_library_name           
-      - name: type                           
-      - name: context_app_version            
-      - name: user_actual_role               
-      - name: context_page_url               
-      - name: context_os_name                
-      - name: context_page_title             
-      - name: anonymous_id                   
-      - name: context_app_build              
-      - name: context_page_search            
-      - name: context_library_version        
-      - name: context_ip                     
-      - name: context_useragent              
-      - name: context_app_name               
-      - name: context_locale                 
-      - name: context_screen_density         
-      - name: context_page_path              
-      - name: context_os_version             
-      - name: category                       
-      - name: duration                       
-      - name: num_of_request                 
-      - name: max_api_resource_size          
-      - name: longest_api_resource_duration  
-      - name: user_id                        
-      - name: root_id                        
-      - name: post_id                        
-      - name: sort                           
-      - name: team_id                        
-      - name: version                        
-      - name: keyword                        
-      - name: count                          
-      - name: gfyid                          
-      - name: field                          
-      - name: plugin_id                      
-      - name: installed_version              
-      - name: group_constrained              
```

Comment:
`group_constrained` looks to be used by looker, so by the description above "using only columns needed by looker." - should be kept?

Reply:
sorry, it exists in LookML, but the field is `null`, it will be removed from the LookML in the following PR, its not used in performance_events dashboard.

Reply:
no, this is exclusively for `performance_events` table. This table is created with the filter `where category = 'performance`` in the below models, the profiler was applied on the final performance_events table, so this column can be dropped in this particular case.